src/explore-education-statistics-frontend/src/components/Feedback.tsx

```diff
@@ -0,0 +1,189 @@
+    setShowNotUsefulForm(false);
+    setShowProblemEncounteredForm(false);
+    setShowBanner(true);
+  };
+
+  const showThankYou = () => {
+    setResponse('');
+    setShowFormSection(false);
+    setShowNotUsefulForm(false);
+    setShowProblemEncounteredForm(false);
+    setShowThankYouMessage(true);
+    setShowBanner(true);
+  };
+
+  return (
+    <div
+      className={classNames(
+        'govuk-width-container',
+        styles.responsiveContainer,
+      )}
+    >
+      {showBanner && (
+        <div className={styles.feedbackBanner}>
+          {showThankYouMessage && (
+            <div className={styles.feedbackPromptQuestions} role="alert">
+              <h3 className={styles.heading}>Thank you for your feedback</h3>
+            </div>
+          )}
+
+          {!showThankYouMessage && (
+            <>
+              <div className={styles.feedbackPromptQuestions}>
+                <h2 className={styles.heading}>Is this page useful?</h2>
+                <Button
+                  className={styles.button}
+                  variant="secondary"
+                  onClick={handleUsefulFeedback}
+                >
+                  Yes <VisuallyHidden>this page is useful</VisuallyHidden>
+                </Button>
+
+                <Button
+                  className={styles.button}
+                  variant="secondary"
+                  onClick={showNotUseful}
+                >
+                  No <VisuallyHidden>this page is not useful</VisuallyHidden>
+                </Button>
+              </div>
+
+              <div className={styles.feedbackPromptQuestions}>
+                <Button
+                  className={styles.buttonReport}
+                  variant="secondary"
+                  onClick={showProblemEncountered}
+                >
+                  Report a problem with this page
+                </Button>
+              </div>
+            </>
+          )}
+        </div>
+      )}
+
+      {showFormSection && (
+        <div className="govuk-!-padding-left-6 govuk-!-padding-right-6">
+          <h3 className="govuk-!-margin-top-5">
+            Help us improve Explore education statistics
+          </h3>
+          <p>
+            Don't include personal or financial information like your National
+            Insurance number or credit card details.
+          </p>
+
+          <FormProvider>
+            {({ formState }) => {
+              const handleSubmit = async (values: FeedbackRequest) => {
+                await feedbackService.sendFeedback({
+                  url,
+                  userAgent,
+                  response,
+                  context: values.context,
+                  issue: values.issue,
+                  intent: values.intent,
+                });
+
+                showThankYou();
+              };
+
+              return (
+                <Form id="feedbackForm" onSubmit={handleSubmit}>
+                  <FormFieldTextArea<FeedbackRequest>
```

Comment:
Not necessarily a big deal but would be better to write a new `FeedbackFormValues` type with the three fields to align more with how we typically declare forms.

I suspect we'll also need this when we actually write the client-side validation (as mentioned in other comment).

Same applies to other fields below.